actions.js

```diff
@@ -8,20 +8,26 @@ window.addEventListener("load", function () {
   let happinessStat = document.getElementById("happinesStat");
   let sleepStat = document.getElementById("sleepStat");
   let energyStat = document.getElementById("energyStat");
+  
 
   window.updateFrontendStatus = function () {
     hungerStat.textContent = pet.hunger;
     happinessStat.textContent = pet.happiness;
     sleepStat.textContent = pet.sleep;
     energyStat.textContent = pet.energy;
+  
+    if(hungerStat.textContent > 80){
+      document.getElementById("hungerchangearrow").classList.replace("bg-green-100","bg-red-100");
```

Comment:
It's recommended to do camelcase for id attributes so instead of `hungerchangearrow` `hungerChangeArrow`

same with `greenarrow`